### PR TITLE
Task4

### PR DESCRIPTION
**Pull request by gadjevb**
Task4 is ready for review!
https://gadjevb.github.io/

### CODE REVIEW

File: task4.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+function validateForm() {
+    var firstName = document.forms["Registration"]["firstname"].value;
+	var lastName = document.forms["Registration"]["lastname"].value;
+	var egn = document.forms["Registration"]["egn"].value;
+	var age = document.forms["Registration"]["age"].value;
+	var address = document.forms["Registration"]["address"].value;
+	var password = document.getElementById("pass").value;
+	var passwordCon = document.getElementById("conPass").value;
+	
+    if (firstName == null || firstName == "") {
```
Comment (freekman) [inline task4.js]: major code pulication

Comment (freekman) [inline task4.js]: this task can be solved with one func

**Comment by gadjevb** [comment]
The task4 description doesn't say that I must validate the form in task3, so I left only one field and a function to validate it.

**Comment by freekman** [comment]
LGTM

**Pull request closed without merging**